java/client/src/org/openqa/selenium/devtools/Log.java

```diff
@@ -33,21 +34,23 @@
 
 public class Log {
 
-  private Log() {
```

Comment:
Utility classes (those that only contain `static` helper methods and classes) should have `private` constructors to stop people accidentally attempting to use the static methods as instance methods. Because "people"